\documentclass[11pt, oneside]{article}   	% use "amsart" instead of "article" for AMSLaTeX format
\usepackage{geometry}                		% See geometry.pdf to learn the layout options. There are lots.
\geometry{letterpaper}                   		% ... or a4paper or a5paper or ... 
\usepackage{color}
\usepackage[parfill]{parskip}    		% Activate to begin paragraphs with an empty line rather than an indent
\usepackage{graphicx}				% Use pdf, png, jpg, or epsÂ§ with pdflatex; use eps in DVI mode
								% TeX will automatically convert eps --> pdf in pdflatex	
								
												
\usepackage{amssymb}

\newcommand{\comment}[1]{{\color{red}[\textsl{#1}]}}
\newcommand{\response}[1]{{\color{black}#1}}

\title{Response to reviews of ``Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants'' for \textit{PNAS}}
\author{Juhye M. Lee, John Huddleston, Michael B. Doud, Kathryn A. Hooper,\\Nicholas C. Wu, Trevor Bedford, Jesse D. Bloom}

\begin{document}
\maketitle

We thank the reviewers for the careful reading and suggestions about the manuscript.
In the revised version, we have performed additional experiments and analyses that address all of the reviewer questions and concerns.

\emph{Below, the reviewer comments {\color{blue} are in blue}, and our responses are in black.}

\color{blue}

\subsection*{Reviewer \#1 Comments}

\subsubsection*{Comments:} 
In their manuscript Lee et al. examine the tolerance of an H3 virus HA for single point mutations using deep mutational scanning. Using the data obtained from cell culture experiments the authors then try to understand if there is a correlation between the observed changes in vitro and actual virus evolution in vivo. Finally, they test if there is any correlation between H3 and H1 mutational tolerance and basically find that measurements made with H1 HA are not helpful to predict H3 evolution. The manuscript is written in a relatively confusing way and the experimental setup is problematic. Some of the obtained data also need re-evaluation/experimental validation. 

\response{We are sorry that the reviewer found some parts of the manuscript confusing.
We have revised the parts of the text that the reviewer highlighted, performed additional validation experiments, and clarified the experimental setup.
These modifications should fully address the reviewer's concerns.}

\subsubsection*{Major points} 

1) It is disturbing that there is no preference for having a methionine in the first position of the HA ORF. The authors discuss this and refer to one 2002 paper that shows additional start sites for non-human H2. This is a pretty weak explanation. If this is really the case the authors should actually isolate/rescue these viruses and test their fitness relative to wild type. 

\response{
We have now used reverse genetics to generate viruses carrying a mutation of the initiating methionine to lysine.
These start-codon mutant viruses grow to appreciable titers, suggesting that there is indeed some flexibility in the site of translation initiation in this HA.
These data are in Figure S5 of the revised manuscript.
As we note in the manuscript, this finding is concordant with prior work on suggesting additional translation initiation sites sites in H2 HAs.
Because translation initiation of HA is not the focus of our manuscript, we have not performed further investigation of the exact mechanism or identity of the alternate start sites.
However, the data we have added clearly show that the canonical ATG is not strictly necessary for viral replication.
}

Of note, position 1 for the H1 HA in the supplementary figure is not even shown. 

\response{
The reviewer is correct. 
Figure~S3 shows data for a H1 HA as measured previously by Doud and Bloom (2016), and these prior data are simply presented here in H3 numbering for comparative purposes (the original Doud and Bloom paper used a different HA numbering scheme that makes site-to-site comparisons with the current study difficult). 
The Doud and Bloom study used mutant libraries in which all codons in HA \emph{except} for the one encoding the N-terminal methionine were mutagenized.
Therefore, that study did not estimate the effects of mutations to the first codon, which is why it is not shown in Figure~S3.
We have added a sentence to the legend for this supplementary figure explaining this fact.
}

2) While other important features of the HA like several cysteines seem to be conserved, there is no pressure on other elements which are considered important for viral fitness including the NGT glycosylation site in the beginning of HA1 which is conserved in all group 2 HAs or the adjacent NAT glycosylation site which is conserved in H3, H7, H10 and H15. On the other hand, head glycosylation sites, which are highly variable in nature, seem to be conserved. Again, viruses without these glycosylation sites should be rescued and evaluated for fitness relative to wild type. 

\response{This is a valid point, and we have now also individually evaluated the reverse-genetics titers of three glycosylation site mutants.
Two of the mutants, T24F and T40V, remove the aforementioned highly conserved but mutationally tolerant NGT and NAT glycosylation sites near the start of HA1.
The third mutation, S287A, removes a glycosylation site that is highly variable in nature but that our experiments suggest is under functional constraint.
The new Figure S6 shows that the T24F and T40V mutants grow to titers comparable to wildtype, supporting the conclusion that they are expendable at least in cell culture. 
However, the titer of the S287A is modestly lower than wildtype, supporting the conclusion that this glycan contributes to viral growth in cell culture.

One possible explanation is simply that cell culture does not fully capture the constraints in nature.
But a more interesting possibility is that conservation in nature is not expected to strictly parallel mutational tolerance.
The tendency for a site to change in nature depends on how much selection there is for the mutations.
For instance, Wei et al (\textit{Science Translational Medicine}, 2:24ra21, 2010) have pointed out that glycosylation motifs evolved rapidly in the globular head of HA from human H1 influenza in the period from 1918 to 2008 (seasonal H1N1), but changed very little in the same period in the swine H1 lineages descended from the 1918 virus.
The likely reason is differential strengths of selection: human influenza is under strong pressure from accumulated immune memory, whereas swine influenza is under less immune pressure because pigs are short lived and so are often naive when infected with influenza.
This fact highlights that even mutationally tolerant sites will only change rapidly if there is a benefit to doing so, while even somewhat constrained sites can change if there is sufficient pressure.
A strength of our experiments is that we have controlled the selection pressure (selection only for viral growth), and so can measure mutational tolerance in the absence of the confounding effects of immunity.

In addition to the data in Figure S6, we have added the following text:

\begin{quote}
\textsl{
The disparity between the relative conservation of these sites in nature and their mutational tolerance in our study could be because cell culture does not fully capture the constraints on HA function on nature, or could be because these sites are not under strong immune pressure and so mutations at them are not positively selected in nature.}
\end{quote}
}

3) It is also surprising that the transmembrane domain seems to be highly plastic, a feature that is certainly not present in nature where it is highly conserved within H3 HAs. 

\response{We have validated that one of these mutations in the transmembrane domain, C199(HA2)K, reaches reverse-genetics titers comparable to wildtype.
These data are in the new Figure S7.
In addition, various other studies have found that chimeric HAs with transmembrane domains from completely different proteins can still maintain HA cell surface expression and fusion activity (Melikyan et al 1999 Mol Biol Cell, Schroth-Diez et al 1998 JVI, Dong et al 1992 JVI, Roth et al 1986 JCB).

We posit that the disparity between the conservation of the transmembrane domain in nature and its apparent mutational tolerance in our experiments could arise at least in part from a lack of immune pressure for mutations in nature, as discussed in the response to the previous point.
}

4) Were sequences from passaged viruses (isolated on MDCKs and other cell lines) included in the comparison between in vitro mutagenesis data and natural evolution data? They need to be excluded since their presence might induce a bias towards what is tolerated/preferred in cell culture. Only sequences obtained directly from patient specimens should be used for the comparison. 

\response{
This is an excellent question.
We now a separate analysis of the relationship between our measured mutational effects and maximum frequency in nature for mutations from unpassaged isolates only.
Because sequences from unpassaged isolates have only become common relatively recently, we could perform this analysis only for the post-Perth/2009 partition of the H3N2 HA phylogenetic tree.
The new Figure 5 shows that the correlation between our measurements and natural evolution is actually \emph{stronger} when we look only at unpassaged isolates.
This makes sense---lab passage mutations will not do well in nature, but will be scored favorably in our experiments.
Therefore, it makes sense that our approach works \emph{even better} when the analysis is restricted to unpassaged isolates.
We thank the reviewer for this suggestion, which has strengthened our work.
}

5) It is unclear how the ``rescaling'' is performed and what it exactly does. The authors need to described and justify this better and should also provide the non-rescaled data in a ``sequence logo'' figure for comparison. 

\response{Re-scaling of the preferences is performed by fitting a stringency parameter that is a reflection of how strongly our experimentally determined preferences correspond to the gene's evolution in nature.
This parameter is fit under an experimentally informed codon substitution model (ExpCM) using the phydms program (Hilton 2017 PeerJ).
A stringency parameter greater than 1 indicates that natural selection favors the same amino acids as the experiments but with greater stringency, while a value less than 1 indicates that natural selection does not favor the amino acids with as much stringency as the experiments.
Because selection in the lab often does not fully reflect the stringency of selection in nature and selection across experiments may not be entirely consistent, we re-scale the preferences by this stringency parameter.
Importantly, re-scaling does not affect any of the evolutionary analyses performed in this paper since it is a constant that multiplies all mutational effects, simply corresponding a change in units on the x-axis of Figure~5.
We now include a supplementary panel A to Figure S3 showing the un-scaled Perth/2009 H3 HA preferences.
}

6) The authors state that it there are more reports of escape from stalk antibodies for H3 than H1. This is simply not true. Actually, the first reference used as evidence for easy selection of H3 anti-stalk escape mutants (Chai et al) reported three escape mutants which a) did not disrupt binding completely and b) were crippled in vitro and in vivo. The authors should remove their statement as there is not systematic data to support this. 

\response{We have removed this sentence from the paper.}

\comment{We looked at this pretty carefully, and there are definitely more reports of selecting anti-stalk antibodies for H1 and H3.
The reviewer's counterpoint is to mention on of the H3 studies. 
However, we didn't say that there are no H3 reports, just that there are a lot more H1 reports.
Certainly this is not systematic proof, but our statement in the text goes to great pains to caveat that this is purely circumstantial and not systematic proof.
But since this isn't a central claim of the paper of anyway, let's just remove it in a concession to the reviewer.
The study we're helping Nicholas Wu with will eventually systematically test this point.
}

7) The biggest issue with the manuscript is the experimental set up. The authors titrate their rescued virus using a TCID50 assay on MDCK cells. Based on the titer that they measure, they passage the virus from the initial rescue once onto fresh MDCK cells using a low MOI. After infection, the virus dilution stays on the cells and is then harvested without centrifugation followed by RNA extraction and sequencing. There are many issue with this process. 

\response{The reviewer has a number of questions and suggestions about the experimental design.
Over the last few years, we have performed a number of studies optimizing procedures for deep mutational scanning of influenza virus libraries (see PMIDs 24859245, 25006036, 27271655) during which we have carefully considered all of the experimental parameters that the reviewer discusses.
Below we briefly explain why each of our choices for these experimental parameters were appropriate.
We do not deny that there are slightly different ways that the experiments could be done, but the in end, the ``proof is in the pudding.'' 
In the manuscript, we show that the measurements made using the experimental parameters that we chose lead to useful data as judged by the following criterion:
\begin{itemize}
\item The results from fully independent experimental replicates are highly correlated (Fig.~1D), which would not be the case if the results were dominated by noise.
\item The experimental measurements do a good job of predicting the effects of individually validated mutations, as shown in Figure S8.
\item The experimental measurements can inform substitution models that describe influenza virus's evolution much better than standard models (Table~1).
\item The experimental measurements provide information that can help predict the fates of influenza virus lineages in nature (Fig.~5).
\end{itemize}
We do not claim that other experimental parameter choices might not lead to equivalent or even slightly better results, but we have carefully considered all of our choices and clearly they lead to very good results.

Below we explain our reasoning for each choice about which the reviewer makes suggestions:
}

a) The TCID50 is likely underestimating the number of virus particles with HA genomic segments because not all of them (likely the minority) will be capable of inducting a full infectious cycle (but their presence might still be detected in the cell supernatants). 

\response{The reviewer is certainly correct that TCID50 assays underestimate the number of HA genomic segments, as it is well known that the number of physical influenza virions exceeds the number of infectious units by 10-100 fold.
This is why we used an extremely low MOI (0.0035) so that most cells are still infected by no more than one virion.
It is impossible to ever grow a viral stock that is completely free of defective particles.
However, in both our original rescue with helper virus and in the passage, we follow the steps suggested by Carolina Lopez and Scott Hensley (PMID 27047455) to minimize defective particles.
These include low MOI (0.0035 TCID50 per cell is quite low), short growth times to avoid accumulation of defective particles (the helper-virus rescue was allowed to propagate for just 24 hours, and the low-MOI passage for just 48 hours), and replacing the media three hours post-infection to remove residual defective particles.
}

b) The infectious inoculum is never washed off the infected cells meaning that any particles that are transferred from the rescue transfection are going to be detected during sequencing. This might even apply to naked RNPs released from dead cells. 

\response{For biological replicates 1, 2, and 3-2, we replaced the original infectious inoculum with 25 mL fresh IGM three hours post-infection.
For replicate 3-1, we did not change the media. 
However, for this replicate, the original inoculum makes a negligible contribution to the total number of virions that are present at the time of sequencing.
Specifically, in the original inoculum placed on the cells, the titer is 2.5 TCID50 / ul, whereas at the time of supernatant collection for sequencing the titer is 32 TCID50 / ul for unconcentrated virus but 400 TCID50 / ul for the concentrated virus.
Therefore, the original inoculum constitutes just 8\% of the viral titer at the time of collection, and therefore makes a negligible contribution to the sequenced material.
We deeply apologize for not including this crucial step in the Methods, and have updated the Methods to now include completion of this step. 
We have also included in the GitHub repository a PDF of the lab notebook describing the rescue and passage of the mutant virus libraries at %\url{https://github.com/jbloomlab/Perth2009-DMS-Manuscript/blob/master/lab_notes/Perth09%20HA%20Library%20Rescue.pdf} (relevant details of the notebook can be found in pages 20-25).
}

c) This would actually not be an issue, if the dilution from the rescue culture to the new culture would be large enough. But this is not the case. The authors passaged $9\times10^5$ TCID50 (and likely much more DIs/RNA) from the rescue cultures onto the new cells in a volume of 25 ml. This is, depending on experiment, between 10 and 1 ml of supernatant of the original rescue and leads to a 1:2.5 or 1:25 dilution respectively. 

\response{We passaged for each mutant virus library $9\times10^5$ TCID50 total across 15 $\times$ 15-cm dishes.
Because we used an inoculum of 2.5 TCID50 / $\mu$l across these fifteen dishes, this corresponds to dilutions that range from 1:37 to 1:294.
The quantity of dishes used to passage each mutant virus library is now mentioned in the Methods.
}

d) The supernatant is never centrifuged before RNA extraction. Cell debris is not removed which is likely contributing significantly to RNA sequences detected in the supernatant. However, RNA associated with cell debris is not representative of RNA from fit, replicating viruses. 

\response{We clarified the viral supernatant before RNA extraction by centrifuging the supernatant at 2000 $\times$ \textit{g} for five minutes to remove any cellular debris.
We failed to mention this step in the Methods, and again deeply apologize for this critical oversight.
We have updated the Methods to now include mention of this step.
}

The experiments should be repeated with at least one more passaging step, proper removal of inoculum and washing of the cells after infection. Centrifugation steps should also be included between the passages and before RNA extraction in the end. 

\response{For the reasons described above, it is not necessary to totally repeat the experiments.
Certainly other experimental parameter choices could have been used, but this is always the case for complex experiments.
Our choices are reasonable, and the four lines of evidence listed at the beginning of this comment show that they yield reproducible and meaningful results.}

\subsubsection*{Minor points} 

1) Many abbreviations are not defined in the manuscript including MDCK-SIAT1, TMPRSS2, MOI, DMEM, FBS, LB. 

\response{We now explain these abbreviations in the manuscript.}

2) The authors state on page 5 that mutations detected in cell culture as deleterious rarely reach high frequency in nature. Could you please give a few examples? 

\response{We individually validated viral growth of a C52A mutant, which removes a Cys critical in forming a disulfide bond with Cys-277 and maintaining HA structure.
We confirmed that this mutant grows very poorly when rescued, and mutations at site 52 never arise at a detectable level.

As another example, mutation N170H fails to reach $>$1\% frequency in nature, and according to our dataset, is a mutation towards a highly unpreferred amino acid. 
}

\comment{Not obvious how to give an example, as there are many mutations we identify as deleterious and many mutations that don't reach high frequency. 
But I suppose if we validate some specific mutation (e.g., a cysteine mutation) as deleterious, we can point out that it doesn't reach high frequency...}

3) The authors picked 31 random clones to evaluate the mutation rate per clone. These sequences should be made available. Did this set include viruses without a proper start codon (see above)? What was their growth phenotype in vitro? 

\response{We have now added the list of mutations from the 31 clones to Dataset S3.
Two of the clones had a mutation that removed the ATG start codon.
One of these resulted in an M(-16)V mutation, while the other resulted in an M(-16)Q mutation which was linked to a Q80I mutation.
According to our preferences, Val and Gln are not very preferred at this site, suggesting that these mutants would not grow well compared to wildtype.
}

\subsection*{Reviewer \#2 Comments} 

\subsubsection*{Comments:} 
The work presented in this manuscript is a thorough analysis of the tolerance of an H3 hemagglutinin to all possible single amino-acid mutations. H3 evolution is of particular relevance now in light of the most recent flu season where vaccine efficacy against H3N2 was estimated at only 25\%. The experimental setup presented in the manuscript is sound and there is strong correlation among the biological and technical replicates. The authors show that data from deep mutational scanning is better able to describe the evolution of H3 HA compared to conventional models. This is consistent with the results of their previous study using deep mutational scanning on the H1 HA of WSN. It is important to note that their approach is ``systematic and quantitative'' in capturing potential effects of mutations in the H3 HA on viral fitness in cell culture. The significance statement is accurate in describing the conclusions of the paper and is understandable to a general scientific audience. 

\response{We thank the reviewer for an accurate and fair summary of our work.}

\subsubsection*{Major points:} 

The authors themselves acknowledge the most important limitation of this study: that the experimental measure of viral growth in cell culture is not reflective of true fitness in nature. Most importantly, the work presented here does not capture host-induced pressures on the antigenic regions of the viral protein. This is evidenced by the discrepancy between the high mutational tolerance they calculated for the stalk region and the actual change in stalk amino acid sequence. A simple protein BLAST comparison between the amino acid sequences of HA from H3N2 viruses Perth09 and HK68 shows a 77\% conservation in head amino acids but a 94\% conservation in stalk amino acids. Additionally, according to figure 2, there is no observable reversion of the G78D mutation they introduced despite the dominance of G78 in the population as seen in figure S1. The authors recognize this in their work and do not attempt to exaggerate the significance of these findings. Still, it would be useful to differentiate between head and stalk mutations when depicting individual amino acid mutations in figure 4 and their frequency trajectories to highlight that the predominance of mutations that occur in nature are in the head. 

\response{We thank the reviewer for acknowledging our efforts to fairly present the considerable caveats associated with cell-culture measurements.

The suggestion to make a version of Fig.~4 that distinguished between the stalk and the head is a good one.
It is difficult to also overlay the structural location (stalk versus head) on the current Fig.~4, since colors are already used to indicate mutational effects.
We have therefore created a new supplemental figure (Fig.~S8) that shows these data---as the reviewer suspected, this shows that head mutations are much more common.

The reviewer is correct that G78D appears to be a site where the correspondence between selection in our experiments and frequency in nature is low.
This makes sense, as G78D is one of the two sites that we specifically identified as being a lab adaptation (see Figure~S1).
We now explicitly mention this fact in the Discussion as follows:

\begin{quote}
\textsl{Indeed, a vast amount of work in virology has chronicled the ways in which experiments can select for lab artifacts or fail to capture important pressures that are relevant in nature.
As an example of this, although we identified G78D as a lab adaptation mutation and the wildtype Gly is not highly preferred at this site, this mutation never reaches above 20\% frequency in nature. }
\end{quote}
} 

The data regarding greater mutational tolerance in the stalk compared to the head should be paired with data from deep mutational scanning utilizing stalk- and head-specific antibodies as a selection process as this group has previously done with WSN H1 (Doud et al., Nat Comm 2018, Doud et al., PLoS Path 2017) to see if tolerant sites are responsible for viral escape. 

\response{We agree that this would be a very interesting experiment to perform.
However, we believe such a study, which would require rigorous examination of escape from multiple head- and stalk-specific antibodies, is beyond the scope of this manuscript.
}

Additionally, the authors should directly show the discrepancies between this in vitro selection and the H3 HA's evolution in nature. This group has performed such an analysis in their previous deep mutational scanning work with the H1 HA of WSN (Doud and Bloom, Viruses 2016). Maximum likelihood phylogenetics can be used to estimate the difference between experimental amino acid preference and natural preference. One might expect that this would provide some insight into the nature of the bias introduced through cell culture. 

\response{This is a good suggestion and we have now added the results of this analysis in Figure S4.
As the logo plot shows, although there are some differences, at most sites there is little difference between the amino acids that are preferred in our experiments to their frequencies in nature.
}

The authors are upfront about the limited degree to which their calculated mutational effect and the maximum mutation frequency in nature correlate, describing it as ``modest.'' One of the major points of the paper is that conclusions from this type of analysis cannot be generalized when using data from strains that are too divergent. The authors should address the possibility that data from deep mutational scanning can be generalized to divergent strains if the study is done in a system that is more reflective of nature. 

\response{We now have acknowledged this possibility in the Discussion in the sentence ``We suspect that performing similar experiments using more realistic and complex selections (e.g., ferrets or primary human airway cultures) might further improve their utility and possibly their generalizability to more divergent strains."}

The experimental approach of deep mutational scanning has been applied to viral proteins before, including HIV env, influenza H1 HA, and NP. The novelty in this paper is that it is the first time such an analysis has been performed for H3 specifically and that it demonstrates the (albeit limited) utility of deep mutational scanning in predicting the success of viral strains in nature. Ultimately, it is clear from the discussion that the authors see this study as an intermediate step, suggesting that their work is "an improvement over no information at all." While something is indeed better than nothing, in my opinion, this paper represents only an incremental step forward in the field of influenza evolutionary forecasting due to the aforementioned limitations, especially since this experiment and subsequent analysis could be performed using more stringent selection methods during viral passaging like treatment with human sera for data that is presumably more faithful to nature. 

\response{This is a fair assessment by the reviewer.
However, this is the first paper to show that data from deep mutational scanning can actually reflect evolutionary outcomes in nature, and provides some insight into the extent by which functional constraints on HA shape its evolution. 
Furthermore, it is possible that this work can provide a useful framework for applying deep mutational scanning studies in the context of evolution in nature.
}

\subsubsection*{Minor Points:} 

The correlation for prospective analysis (post-Perth) is smaller than the correlation for post-hoc analysis (pre-Perth). Since prospective analysis is more important when predicting evolutionary fates, it might be beneficial for the authors to comment on this difference. 

\response{We now include an analysis of mutational effects and maximum frequency for mutations of unpassaged isolates (see Figures 5, 6, and S7) from the post-Perth partition of the phylogenetic tree to limit any potential biases that our experimental measurements may have towards mutations from cell- or egg-passaged isolates.
The results show that this correlation is stronger when considering mutations from unpassaged isolates in a post-hoc analysis, demonstrating the potential utility of our experimental measurements in a prospective analysis, specifically when passaged isolate sequences are excluded.
}

A number of issues interfere with the clarity of this manuscript, particularly when appealing to a broader scientific audience. While this is understandable given the nature of this study, several adjustments should be made. A clear description of mutational effect is never provided in the results, though it forms a crucial component of the data. As such, it is not really intuitive as to why the range from -10-0 should be considered "neutral." Additionally, mutational tolerance is quantified as Shannon entropy of the re-scaled amino-acid preferences at each site, but it is unclear how this entropy is calculated. The entire first half of the second results section describing how the experimentally informed codon substitution model they generate is better than existing models and the associated Table 1 is very difficult to understand. AIC is never explained. The significance of dN/dS being much less than 1 for conventional models should be explained. The purpose of modeling of ExpCM, site avg. is unclear. 

\response{We appreciate these suggestions from the reviewer and have made the following changes:
\begin{itemize}
\item We have now included a description of how we calculated mutational effects in the Results section.
\item We agree that a cutoff of -10 to 0 is arbitrary and removed the sentence ``Mutations that reach high frequencies generally have neutral or beneficial effects according to our experimental measurements." 
\item We included in the Methods a description of how we calculated Shannon entropy.
\item We now define AIC in the legend of Table 1.
\item We have now included an explanation of dN/dS $\ll$1 for conventional substitution models, and now describe the utility of building an ExpCM, site avg. substitution model
\end{itemize}
}

In figure 4, it would be useful to show Perth09 on the tree and mention that closely related strains are excluded from analysis in the figure legend. As it stands, there is just an unlabeled cluster of viruses for which no individual amino acid mutations are characterized. It is also important for the authors to clearly state what frequency they are talking about. Specifically, in consideration of a particular mutation, what is the population in the denominator? 

\response{We have now labeled the Perth/2009 strain on the tree in Figure 4 and now mention in the caption that we have excluded closely related strains from the analysis...
}

\comment{Fixed the figure. Maybe John and Trevor can do a better job than me at explaining the frequency denominator. Is it the population observed at a given pivot?}

The graph labels in figure 8B could be cleaner. The question marks after "absolutely conserved" and "clade-specific" are extraneous. Indeed, the graph labels themselves ("HA domain," "absolutely conserved? ""clade-specific?") are unnecessary. 

\response{We have now fixed the graph labels in figure 8B accordingly.}




\end{document}  